content/guides/getting-started/box-glossary.md

```diff
@@ -0,0 +1,92 @@
+| --- | --- | --- |
+| Admin | Superuser, administrator, supervisor | The main admin on an enterprise account. Admins can manage users and groups, view and edit all of their organization’s files and folders, log in to any user’s account within their organization, edit settings for their organization, and run or access reports. The Box Admin is the principal account administrator. |
+| Admin Console | | The dashboard area for administering a particular enterprise. Accessible by the admin, group admins and co-admins. |
+| Anonymous User | | A user that is not logged in. |
+| App User | | [App users][app user] are only accessible with the API - they do not have login credentials. They can be created by a Service Account and are only applicable to applications leveraging server to server authentication. App Users are tied to the application used to create them, and the user itself cannot be moved under another application. They can however collaborate on content outside of the application. |
+| Bookmark / Web Link | Symbolic Link, Symlink, Soft Link, Reference, Relationship | |
+| Box App Center | App Store | The first place for Box users to find out about applications that they can use in combination with Box. |
+| Box Command Line Interface | [Box CLI][CLI] | User-friendly command line tool allows both technical and non-technical users to leverage the Box API to perform routine or bulk actions. |
+| Box Custom Skill | Custom Skill, Box Skill | An application that performs custom processing for files uploaded to Box. Skills are designed to make it possible to use third-party Machine Learning services to automatically extract information from files uploaded to Box. |
+| [Box Embed][embed] | | An HTML-based framework that makes it possible to embed the Box Web App experience anywhere in the 3rd party applications. Box Embed provides the ability to upload, search, comment, share, tag, and edit files using Box Edit. |
+| Box Platform | | An API-driven cloud content management and collaboration platform that provides developers with the tools and infrastructure to build custom applications, integrate workflows, and leverage powerful features for secure document storage, access control, versioning, metadata management, and real-time collaboration. |
+| [Box Sample Code Catalog][code-catalog] | | Allows developers and admins to search through code repositories of tools, automation scripts, self paced workshops, and demo apps in multiple programming languages. The catalog contains more that eighty open source repositories and guides. |
+| Box Shield | | A security solution offered by Box that helps protect sensitive data, prevent unauthorized access, and detect potential threats through automated classification and proactive monitoring. |
+| Box Skills | | Box Skills are AI-powered capabilities that enhance the functionality of Box by automatically extracting insights and metadata from content. |
```

Comment:
I'm wondering if we should explain the difference between these Box Skills and Box (Custom) Skills.